### PR TITLE
add: Set Hellonext Wallpapers

### PR DESCRIPTION
**Pull request by imkarthikk**
## Description

Raycast extension to set wallpapers from Hellonext.

## Screencast

Metadata-2
Metadata-3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/hellonext-wallpapers/package.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  ],
+  "preferences": [
+    {
+      "name": "applyTo",
+      "title": "Set Wallpaper To",
+      "description": "Set the wallpaper to which screen.",
+      "type": "dropdown",
+      "data": [
+        {
+          "title": "Active Desktop",
```
Comment (pernielsentikaer) [inline extensions/hellonext-wallpapers/package.json:35]:
```suggestion
          "title": "Active Monitor",
```

File: extensions/hellonext-wallpapers/package.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          "value": "current"
+        },
+        {
+          "title": "All Desktops",
```
Comment (pernielsentikaer) [inline extensions/hellonext-wallpapers/package.json:39]:
```suggestion
          "title": "All Monitor",
```

File: extensions/hellonext-wallpapers/package.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      "name": "set-hellonext-wallpaper",
+      "title": "Set Hellonext Wallpaper",
+      "description": "Set a wallpaper from Hellonext as your desktop wallpaper.",
+      "mode": "view"
+    },
+    {
+      "name": "random-hellonext-wallpaper",
+      "title": "Random Hellonext Wallpaper",
+      "description": "Set a random wallpaper from Hellonext as your desktop wallpaper.",
+      "mode": "view"
```
Comment (pernielsentikaer) [inline extensions/hellonext-wallpapers/package.json:24]: Could be a [no-view](https://developers.raycast.com/information/manifest#command-properties) instead, and use a HUD to show a success message instead.

File: extensions/hellonext-wallpapers/package.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  ],
+  "preferences": [
+    {
+      "name": "applyTo",
+      "title": "Set Wallpaper To",
+      "description": "Set the wallpaper to which screen.",
```
Comment (pernielsentikaer) [inline extensions/hellonext-wallpapers/package.json:31]:
```suggestion
      "description": "Set the wallpaper to which monitor.",
```

File: extensions/hellonext-wallpapers/src/set-hellonext-wallpaper.tsx [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+        }
+        return;
+      }
+    }
+
+    _fetchWallpaper().then();
+  }, []);
+
+  return (
+    <List isShowingDetail={true} isLoading={raycastWallpaper.length === 0} searchBarPlaceholder={"Search WallPapers"}>
```
Comment (pernielsentikaer) [inline extensions/hellonext-wallpapers/src/set-hellonext-wallpaper.tsx:43]:
```suggestion
    <List isShowingDetail={true} isLoading={raycastWallpaper.length === 0} searchBarPlaceholder={"Search wallpapers"}>
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @imkarthikk 👋 

Thanks for your contribution 🔥 

I have some feedback and suggestion for you to address:

- It looks like you only need one file in assets, could you remove the rest?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by imkarthikk** [comment]
I'm on it @pernielsentikaer

**Comment by imkarthikk** [comment]
@pernielsentikaer Done Peter. Fingers crossed. 👀😍

**Pull request merged**